include/simdjson/generic/ondemand/json_iterator-inl.h

```diff
@@ -14,6 +15,7 @@ simdjson_really_inline json_iterator &json_iterator::operator=(json_iterator &&o
   token = other.token;
   parser = other.parser;
   _string_buf_loc = other._string_buf_loc;
+  error = other.error; // For some reason this was not included initially? This made the error code sticky.
```

Comment:
Comment is unnecessary: it was simply a bug it wasn't included :)

Reply:
I am removing the comment. I added these comments because I was concerned that there might have been some important logic that I was missing.